Source/DataSources/PolygonGeometryUpdater.js

```diff
@@ -532,12 +540,17 @@ define([
         }
 
         var options = this._options;
-        var positions = Property.getValueOrUndefined(polygon.positions, time, options.polygonHierarchy.positions);
-        if (!defined(positions)) {
+        var hierarchy = Property.getValueOrUndefined(polygon.hierarchy, time, this._scrach);
```

Comment:
It's actually not needed at all.  So I deleted the variable and parameter.  Thanks.  This is ready.